test/functional/resendwallettransactions.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test resendwallettransactions RPC."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+class ResendWalletTransactionsTest(BitcoinTestFramework):
+
+    def __init__(self):
+        super().__init__()
+        self.extra_args = [['--walletbroadcast=false']]
+        self.num_nodes = 1
+
+    def run_test(self):
+        # Should raise a RPC error if walletbroadcast is disabled.
+        assert_raises_jsonrpc(-32600, "Error: Wallet transaction broadcasting is disabled with -walletbroadcast", self.nodes[0].resendwallettransactions)
+
+        # Should return an empty array if there aren't mempool transactions.
```

Comment:
Comment is incorrect. This isn't about transactions in the mempool, it's about transactions in the wallet.